tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -15,19 +15,20 @@
 PERF_TEST_DIFFERENCE_TOLERANCE = 5
 
 
+# retrieves secrets(license info) required for running starccm+ from aws secrets manager stored in dev acc
```

Comment:
Here and below. Comments should be enclosed within triple quotes just after method signature